dts/bindings/spi/nordic,nrf-spim.yaml

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2019 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+title: Nordic nRF Family SPIM (SPI master with EasyDMA) node
```

Comment:
Might be better to have something like

```
title: Nordic nRF family SPIM
description: Nordic nRF family SPIM (SPI master with EasyDMA) node
```

The `description` is what shows up in the comment in `generated_dts_board.h`. `title` is currently unused.

Reply:
Sure. Will switch those.

Reply:
Done.